api/envoy/config/cluster/v3/cluster.proto

```diff
@@ -603,6 +603,22 @@ message Cluster {
     // This is limited somewhat arbitrarily to 3 because prefetching connections too aggressively can
     // harm latency more than the prefetching helps.
     google.protobuf.DoubleValue prefetch_ratio = 1 [(validate.rules).double = {lte: 3.0 gte: 1.0}];
+
+    // Indicates how many many streams (rounded up) can be anticipated across a cluster for each
```

Comment:
Can you expand here on how this differs from `prefetch_ratio`? I think the intuition is that rather than prefetch a connections for every single upstream endpoint, we do it with some lookahead based on LB. For large clusters and low QPS, this is a big payoff.